### PR TITLE
Update spell-checking.md

### PR DESCRIPTION
**Pull request by dm5n**
Minor spelling correction: Yuo → You

### CODE REVIEW

**Comment by nathanlesage** [comment]
There are some merge conflicts; so far the files look good but it would be awesome if you could fix these merge conflicts so I can actually merge it! :)

**Comment by dm5n** [comment]
Ok, appears fixed.

File: docs/en/academic/custom-templates.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # Custom Templates
 
+
```
Comment (nathanlesage) [inline docs/en/academic/custom-templates.md:3]: Superfluous linebreak

File: docs/en/core/writing-modes.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Writing Modes
+In the View Menu, Zettlr features several Editor settings that may enhance your writing experience. Have a play and see what works for you!
```
Comment (nathanlesage) [inline docs/en/core/writing-modes.md:2]: Make sure to add an empty line in between headings and the text

File: docs/en/core/writing-modes.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+## Night Mode
+Also commonly known as 'Dark Mode', clicking this option will change the background colour of the Editor to black and the default text colour to white.
```
Comment (nathanlesage) [inline docs/en/core/writing-modes.md:5]: We should call that Dark mode consistently, I have to fix that in the translations sometime (very) soon!

File: docs/en/core/writing-modes.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+## Distraction-Free Mode
+The distraction free mode removes visual clutter from the Editor environment. Zettlr opens with the [File Manager sidebar](file-manager.md) on the left side of the screen, showing your Workspaces and Files. If you now click on 'Distraction free' in the View menu, you will notice the sidebar disappear and your text content shift towards the centre of the screen.
+
+In this mode, the sentence you're currently working on is highlighted and completed sentences are dimmed. Clicking anywhere on a sentence, shifts the cursor position and highlights that sentence.
```
Comment (nathanlesage) [inline docs/en/core/writing-modes.md:10]: The dimming is an option that can be disabled

File: docs/en/get-involved.md [hunk 5]
```diff
@@ -6,14 +6,16 @@ Do you want to make Zettlr an even better app? That's great! Whether you are a u
...
 * [The official Twitter account](https://www.twitter.com/Zettlr) — Here you can follow the updates to the app in real time. It's the only place where we regularly announce everything we do. Additionally, we engage in most discussions, so if you have questions, you can drop them here.
 * [The official Facebook page](https://fb.me/Zettlrapp) — Here you can message us, if you don't have a Twitter account.
-* [Our YouTube channel](https://www.youtube.com/c/Zettlr) — If you are more the visual type, you can find some introductory videos here.
+* [Our YouTube channel](https://www.youtube.com/c/Zettlr) — If you are more the visual type, you can find some introductory videos her
+* [Zettlr user forum](https://forum.zettlr.com) — (Deprecated)
 * [The GitHub issue tracker](https://github.com/Zettlr/Zettlr/issues) — This is the core of our endeavours to make the app better and better. If you spot a bug, have a suggestion or want to propose a feature, here's the right place. Nevertheless, especially when it comes to questions regarding the user workflow or new features, **it's best to discuss your ideas on the GitHub or on Reddit first**.
 
+
```
Comment (nathanlesage) [inline docs/en/get-involved.md:18]: Superfluous line break

File: docs/en/reference/settings.md [hunk 6]
```diff
@@ -85,14 +85,15 @@ The final setting allows you to choose whether you want to type from right to le
 
 ![settings_export.png](../img/settings_export.png)
 
-The export tab allows you to tweak preferences concerning how your files are exported. First, you will have a few checkboxes and options available:
+
```
Comment (nathanlesage) [inline docs/en/reference/settings.md:88]: Superfluous linebreak

File: docs/en/reference/settings.md [hunk 7]
```diff
@@ -85,14 +85,15 @@ The final setting allows you to choose whether you want to type from right to le
...
 * **Remove tags from files**: If checked, Zettlr will run a special Lua-filter that removes tags which are defined within your files (not the ones in the YAML frontmatter)
 * **Internal Link Handling**: With the radio buttons you can choose what happens during export with internal links in double-square-brackets: Should they be completely removed, just unlinked (= remove the square brackets), or left alone?
-
 The **target directory** controls where Zettlr will put files after the exporter ran. If you choose the temporary directory, your files will not clutter your computer. This makes it easy to export a file several times and tweak, e.g., the layout of the file. Once you are happy with the results, you can simply save the file to another place (because your operating system will clean out the temporary directory without warning!). If you choose the current working directory, Zettlr will place the exported file next to your source Markdown file. Careful not to overwrite files!
 
+
```
Comment (nathanlesage) [inline docs/en/reference/settings.md:96]: Superfluous linebreak

File: docs/en/reference/settings.md [hunk 8]
```diff
@@ -133,8 +134,10 @@ The last field in the Zettelkasten tab in the preferences window concerns the wa
 
 Always remember to adapt your regular expression so that it matches what the ID generator spits out. To successfully identify the ID with the pattern, `%Y-%M-%D_%h:%m:%s`, Zettlr would need the following regular expression: `(\d{4}-\d{2}-\d{2}_\d{2}:\d{2}:\d{2})`. If you use `%uuid4`, you can use the following RegEx: `[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-4[a-fA-F0-9]{3}-[89aAbB][a-fA-F0-9]{3}-[a-fA-F0-9]{12}`.
 
+
```
Comment (nathanlesage) [inline docs/en/reference/settings.md:137]: Superfluous line break

File: docs/en/reference/settings.md [hunk 9]
```diff
@@ -133,8 +134,10 @@ The last field in the Zettelkasten tab in the preferences window concerns the wa
...
 #### Other Options
 
+
```
Comment (nathanlesage) [inline docs/en/reference/settings.md:140]: Superfluous linebreak

File: docs/en/reference/settings.md [hunk 10]
```diff
@@ -153,10 +156,15 @@ This tab controls how the editor displays certain elements. Zettlr uses a semi-p
...
+#### Theme
+
+Next you can choose which theme to use. Currently, you have five to choose from. Each theme comes with a little explainer to tell you how it looks. But don't hesitate: Click yourself through until you find one that you like!
+
+Click on any of the previews to switch your application to that theme.
 
 Furthermore you can constrain the size of images, which is especially beneficial if you use a lot of portrait oriented images. The first slider controls the maximum amount of width images are allowed to occupy. Setting this to 50%, for instance, will keep images at half the width of the text body. 100% disables this (images will never be wider than 100% of the text body width). The second slider controls the maximum height of the images relative to the window size. 50 percent would mean that images will never be higher than half of the window's total size. 100 percent disables this behaviour. Unlike the width, images may very well exceed 100 percent of the viewport if they are large and thin.
 
+
```
Comment (nathanlesage) [inline docs/en/reference/settings.md:167]: Superfluous line break

File: docs/en/reference/settings.md [hunk 11]
```diff
@@ -183,7 +191,15 @@ The last thing you can tweak is the replacement table : These are simply charact
 
 The advanced tab contains options that are helpful for advanced tasks.
 
-The “pattern for new filenames” is what the textfield will look like when you create a new file. It will be prepopulated by whatever you type here. You can make use of a few variables (the ID following the pattern you have selected in the Zettelkasten tab, year, month, day, hour, minute and second). If you tick the checkbox, Zettlr will not even ask you for a filename when you create a new file, which might speed up your workflow.
+
```
Comment (nathanlesage) [inline docs/en/reference/settings.md:194]: Superfluous line break

File: docs/en/reference/settings.md [hunk 12]
```diff
@@ -183,7 +191,15 @@ The last thing you can tweak is the replacement table : These are simply charact
...
+The “pattern for new filenames” is what the textfield will look like when you create a new file. It will be prepopulated by whatever you type here. You can make use of the variables below (the ID following the pattern you have selected in the Zettelkasten tab, year, month, day, hour, minute and second). If you tick the checkbox, Zettlr will not even ask you for a filename when you create a new file, which might speed up your workflow.
+
+
+The **debug option** is used to control whether or not you are able to "reload" the graphical user interface (GUI) by pressing `F5` and display the Chrome developer tools to inspect what the app is doing. Naturally, you would want to do this as a developer.
+
+You can also help Zettlr by **subscribing to beta releases**. When you tick this option, Zettlr will notify you about new beta releases in addition to normal releases. You can then decide whether to use the beta version, or not. Activate **RMarkdown file support** to enable the detection of `.rmd` files used for R-Markdown. It is inactive by default, as some other programs (for instance, by Adobe) also use this extension to store binary data, which would cause problems.
```
Comment (nathanlesage) [inline docs/en/reference/settings.md:200]: RMarkdown file support is not present in the 2.0 options anymore, it's baked in by default

File: docs/en/reference/settings.md [hunk 13]
```diff
@@ -183,7 +191,15 @@ The last thing you can tweak is the replacement table : These are simply charact
...
+
+The text area to the right controls what files will be displayed in the **attachment** sidebar. It is simply a comma-separated list of all file extensions you would like to have at your disposal. The initial list should contain most extensions you might be using.
```
Comment (nathanlesage) [inline docs/en/reference/settings.md:202]: Renamed to "other files" in lieu of a better name (they weren't attachments, and other files may sound awkward but better than nothing)

**Review by nathanlesage — CHANGES_REQUESTED** [review]
Looks good! A few stylistic things but nothing substantial to criticize :)

File: docs/en/academic/custom-templates.md (hunk 1 above)
Comment (dm5n) [reply docs/en/academic/custom-templates.md:3]: Changed

File: docs/en/core/writing-modes.md (hunk 3 above)
Comment (dm5n) [reply docs/en/core/writing-modes.md:5]: Should change this in docs after you change in the app?

File: docs/en/core/writing-modes.md (hunk 2 above)
Comment (dm5n) [reply docs/en/core/writing-modes.md:2]: Fixed

File: docs/en/get-involved.md (hunk 5 above)
Comment (dm5n) [reply docs/en/get-involved.md:18]: ☑️

File: docs/en/reference/settings.md (hunk 10 above)
Comment (dm5n) [reply docs/en/reference/settings.md:167]: ☑️

File: docs/en/core/writing-modes.md (hunk 4 above)
Comment (dm5n) [reply docs/en/core/writing-modes.md:10]: Fixed

Comment (dm5n) [reply docs/en/core/writing-modes.md:10]: Also reworded explanation of Typewriter mode.

File: docs/en/reference/settings.md (hunk 11 above)
Comment (dm5n) [reply docs/en/reference/settings.md:194]: Fixed

File: docs/en/reference/settings.md (hunk 9 above)
Comment (dm5n) [reply docs/en/reference/settings.md:140]: fixed

File: docs/en/reference/settings.md (hunk 12 above)
Comment (dm5n) [reply docs/en/reference/settings.md:200]: removed reference to RMarkdown

File: docs/en/reference/settings.md (hunk 13 above)
Comment (dm5n) [reply docs/en/reference/settings.md:202]: ☑️

File: docs/en/reference/settings.md (hunk 6 above)
Comment (dm5n) [reply docs/en/reference/settings.md:88]: fixed

File: docs/en/reference/settings.md (hunk 7 above)
Comment (dm5n) [reply docs/en/reference/settings.md:96]: fixed

File: docs/en/core/writing-modes.md (hunk 3 above)
Comment (dm5n) [reply docs/en/core/writing-modes.md]: Fixed and renamed to Editor modes

**Review by nathanlesage — APPROVED** [review]

**Comment by nathanlesage** [comment]
Perfect, thank you! :)

**Comment by dm5n** [comment]
Cool 😃

**Pull request merged**